BAZProjectC1/MovieAPI.swift

```diff
@@ -31,5 +31,34 @@ class MovieAPI {
 
         return movies
     }
+    
+    func getMoviesUpdate(completion: @escaping ([Movie]) -> ()) {
+        guard let url = URL(string: "https://api.themoviedb.org/3/trending/movie/day?api_key=\(apiKey)") else {
+            return
+        }
+        
+        let session = URLSession.shared
+        session.dataTask(with: url) { (data, response, error) in
+            if let data = data {
```

Comment:
Podemos cambiar este if let por un guard let?
```suggestion
            guard let data = data else { return }
```